dependencies/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
I think we should have an ArgoCD application set for each directory under the dependencies directory.
There is no need to deploy the `registry` component.

Reply:
I have placed the ArgoCD application set for each component in `argo-cd-apps/base/member/konflux-ci/`. Do I need to move it?